CHANGELOG.md

```diff
@@ -11,6 +11,124 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.8.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.7.0. This
+release brings new features such as:
+- A brand new file-based configuration for Ginkgo: you can now
```

Comment:
this is incomplete

Reply:
my bad, context switch at the wrong time